src/utils/utils.cairo

```diff
@@ -236,6 +243,29 @@ namespace Helpers {
         return (bytes_array_len=32, bytes_array=value_as_bytes_array);
     }
 
+    // @notice This function is a variant of `uint256_to_bytes_array` that encodes the uint256 with no padding
+    // @param value: value to convert.
+    // @param idx: index of res array
+    // @param res: resultant encoded bytearray, but in reverse order
+    // @param dest: reversed res, putting byte array in right order
+    // @return bytes array len
+    func uint256_to_bytes_no_padding{bitwise_ptr: BitwiseBuiltin*, range_check_ptr}(
```

Comment:
To reiterate the motivation for this helper:

We encode the salt/nonce in rlp, which means translating it to a byte array when its value is greater than 127.

When I tested the salt in the case 2^55-1, `Helpers.felt_to_bytes_array` was returning a byte array of length one with [2^55-1], downstream causing incorrect rlp encoding than the python implementation.

So I opted to lift the value as a uint256 to handle a greater range accurately and needed to write this helper to get the byte array that took the uint256 value and translated it into a byte array with no padding.